app/components/gist/files_component.rb

```diff
@@ -1,7 +1,7 @@
 class Gist::FilesComponent < ViewComponent::Base
+  attr_reader :gist
+
```

Comment:
view component doctrine suggests to move the attr_readers to the bottom under a `private` tag so it cant be accessed from outside the component 🙈

Reply:
God that's Scheisse